src/server.rs

```diff
@@ -81,6 +81,16 @@ pub fn create_server(conf: ConfigOpts, wallet: Wallet<AnyBlockchain, Tree>) -> S
                     Some(address) => address,
                     None => return Ok(response.body(not_found().as_bytes().to_vec())?),
                 };
+                match is_mine_address(&*wallet, address) {
+                    Ok(mine) => {
+                        if !mine {
+                            return Ok(response.body(
+                                format!("Not mine address {}", address).as_bytes().to_vec(),
```

Comment:
same nit as above.. should read "Not my address: {}" or maybe "Address {} is not mine"

Reply:
thanks